framework/wazuh/core/tests/test_wlogging.py

```diff
@@ -219,3 +219,24 @@ def __init__(self, log_type):
     # Return False
     cf = wlogging.CustomFilter('testA')
     assert not cf.filter(MockedRecord('testB'))
+
+
+@pytest.mark.parametrize('value, expected',
+                         [
+                             ('Example log', True),
+                             ('Wazuh Internal Error', False),
+                             ('WazuhInternalError', False)
+                         ])
```

Comment:
```suggestion
@pytest.mark.parametrize('value, expected', [
    ('Example log', True),
    ('Wazuh Internal Error', False),
    ('WazuhInternalError', False),
    ('WazuhError', True),
    ('InternalError', True)
])
```

Reply:
Fixed in 08f6d1a3a8a5db68c5dddd11bc1b88a047614105.